### ISSUE TITLE
Add TODO:'s in Spade

### ISSUE THREAD

**Issue by Falthazar** [codecadwallader/codemaid#418]
### Environment
- Visual Studio version: 2017 Community
- CodeMaid version: 10.2
- Code language: VB.Net

### Description
I was thinking of a little indicator on the right hand side of Spade to show if there is a TODO in the code item? 
It would help to see which functions still need work, and you could filter by ones with TODOs, or sort by number of TODOs. I don't think it should take over the task list, but when you come to a new project, or going back to an old one that needs work, that might come in handy to see what the task list was.

**Comment by codecadwallader** [comment]
Thanks for the suggestion.  We do have some existing subtle text indications, e.g. 's' for static and 'ro' for readonly that is used for describing properties/methods.  Were you thinking something along those lines if a TODO is detected within a member?  What about TODO statements that are not inside any member (e.g. between methods)?

**Comment by Falthazar** [comment]
Yep! Exactly like that. Just a minor note that indicates the member has a TODO.
Not sure about TODOs between methods though. It could be a little dash or something for where it is between methods? Would definitely need to be toggle-able though.
